src/services/messaging.js

```diff
@@ -16,21 +16,24 @@ async function messageUser(userId, msg) {
   }
 }
 
-async function messageUsers(userIds, msg) {
+async function messageUsers(userIds, msg, giftedId) {
+  msg._id = giftedId;
```

Comment:
This is removing the mongoDB unique id from the message. Shouldn't it be something like `msg.giftedId = giftedId` instead?